frontend/src/components/Home/Home.tsx

```diff
@@ -1,9 +1,44 @@
-import './Home.css';
+import "./Home.css";
+
+import HomeWithoutLogin from "./HomeWithoutLogin/HomeWithoutLogin";
+import axios from "axios";
+import {useEffect, useState} from "react";
+import {User} from "../../Types/User.ts";
+
 export default function Home() {
+    const [userData, setUserData] = useState<User>({
+        id: "",
+        name: "",
+        givenName: "",
+        familyName: "",
+        email: "",
+        picture: ""
+    });
+
+    function loadUser() {
+        axios.get('/api/users/me')
+            .then(response => {
+                setUserData(response.data)
+                console.log("User data loaded:", response.data);
```

Comment:
You should not use console.log in production code. Instead you can use a logger

Reply:
Thanks! I overlooked these console.logs. I replaced them with a Logger